src/gameserverfromfile.py

```diff
@@ -0,0 +1,63 @@
+import sys
```

Comment:
what does `gameserverfromfile.py` do?

Reply:
Instead of generating random deals it loads the deal from a file input.ben - this file can be generated using pbn2ben.py so it is possible to download a pbn-file and play it.

No problem leaving it out

Reply:
I will merge it into gameserver at next PR

Reply:
i think it's a very useful feature.
perhaps it would be cleaner to make the existing gameserver load boards from a file specified by a parameter and default to random hands.
as it is, the two gameservers are almost the same

Reply:
Agree, and I think I will add the PBN-conversion to the server, so It can read a PBN-file with boards, and probably also LIN-files.

One problem with this might be that we always start from the top of the file, so either there should be some feature to select the board, or we should have a gamedb pr. file.